src/dotnet/commands/dotnet-nuget/Program.cs

```diff
@@ -35,9 +35,14 @@ private class NuGetCommandRunner : ICommandRunner
             public int Run(string[] args)
             {
                 var nugetApp = new NuGetForwardingApp(args);
-
+                nugetApp.WithEnvironmentVariable("DOTNET_HOST_PATH", GetDotnetPath());
```

Comment:
ping again @dsplaisted does it look good to you?

Reply:
@wli3 Yes, assuming `GetDotnetPath()` returns the value we want here.